mattumland/Salon
# Feature/art details

## Is this a fix or a feature?
+ Feature

## What is the change?
+ Structures, styles the details view using a hard-coded `selectedArt` object
+ Incorporates `Route` to separate the landing page and details page

## Where should the reviewer start?
+ src/components/ArtDetails directory

## How should this be tested?
+ Run server, visit http://localhost:3000/437329
+ An image and a caption should be visible
+ UI should be responsive to mobile and tablet views
